frontend/packages/dev-console/src/components/pipelines/PipelineRuns.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import PipelineRunsList from '../pipelineruns/PipelineRunList';
+import { pipelineRunFilterReducer } from '../../utils/pipeline-filter-reducer';
+
+const filters = [
+  {
+    type: 'pipelinerun-status',
+    selected: ['Succeeded'],
+    reducer: pipelineRunFilterReducer,
+    items: [
+      { id: 'Succeeded', title: 'Complete' },
```

Comment:
Any reason not to match the title to id?

Reply:
AFAIK this was done according to UX. I can followup with the dev who works on pipelines.
@serenamarie125

Reply:
Abhi confirmed this is intentional after discussing the values with UX and the tekton team

Reply:
yes @spadgett @christianvogt the decision was to match what's in the upstream